laws/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -23,6 +26,15 @@ class LawTests extends FunSuite with Discipline {
   checkAll("String", OrderLaws[String].order)
   checkAll("String", GroupLaws[String].monoid)
 
+  {
+    // TODO: test a type that has Eq but not Order
+    implicit val g: Group[Int] = Group.additive[Int]
+    checkAll("Option[Int]", OrderLaws[Option[Int]].order)
+    checkAll("Option[Int]", GroupLaws[Option[Int]].monoid)
+    checkAll("Option[String]", OrderLaws[Option[Int]].order)
```

Comment:
That's a good idea -- I'll fix the typo for now and open an issue about looking into a macro.